Sources/XcodeGenKit/ProjectGenerator.swift

```diff
@@ -33,7 +33,16 @@ public class ProjectGenerator {
         return spec.configs.first { $0.type == .release }!
     }
 
-    func validate() throws {
+    func getPath(_ path: String) -> Path {
```

Comment:
I just started to add an extension of `Path`, but found out that appending paths already checks for absolute paths, so this was all unnecessary :)